### DISCUSSION TITLE
USDZ model isn't shown if the webpage has password from .htaccess

### DISCUSSION THREAD

**Discussion by GiovanniTurri** [google/model-viewer#2852] [Q&A]
### Description

In my server I created .htpasswd and .htaccess files to protect an HTML page which have inside `<model-viewer ios-src="mymodel.usdz">`, but after the correct "login", in iOS devices (both iOS14 and iOS15) is triggered the error of not support of the model for that OS version (a typical error for unsupported features generated by big user's mistakes).
Removing only the .htaccess file, it works.

On Android, no problem in both cases.
I think there is something on the ios-src that can't handle a login or any type of barriers to access the resource.

Is there something I can do as a workaround (obviously, giving an external path to the usdz file stored in an other server works)?

Thank you so much

### Version
- model-viewer: v1.8.0

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ X] Safari

### OS

- [ ] Android
- [ X] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [ ] WebXR
- [ ] SceneViewer
- [ X] QuickLook

**Comment by elalish** [comment]
This looks to be a Safari/Quick Look bug, rather than anything we can fix in MV. Can you give  a repro  link? I can try  to file a bug on Apple.

**Reply by GiovanniTurri** [reply]
I see.
Yes you can reproduce at http://test.wear-mobile.com/ logging with user: git and psw: hub

**Reply by GiovanniTurri** [reply]
@elalish do you think there can be a way to contact Apple to have more informations about this?
Thank you

**Reply by elalish** [reply]
file a bug: https://bugs.webkit.org/

**Reply by elalish** [reply]
@grorg Here's another Apple bug if you haven't seen it yet.
